tests_integration/scripts/integration_account_setup.sql

```diff
@@ -63,7 +63,7 @@ CREATE OR REPLACE NETWORK RULE snowflake_docs_network_rule
 
 CREATE OR REPLACE SECRET test_secret
   TYPE = GENERIC_STRING
---   SECRET_STRING = ''; -- provide password
+  SECRET_STRING = 'test';
```

Comment:
i'm not sure why this was commented out originally, without this param the query fails